spec/helpers/application_helper_spec.rb

```diff
@@ -161,7 +161,7 @@ def preload_link(url)
 
     context "when dark theme is present" do
       before do
-        dark_theme = Theme.create(
+        _dark_theme = Theme.create(
```

Comment:
Was this intentional?

Reply:
Oh. Is the _ signifying it is an unused local variable?

Reply:
yeah... that var is not used at all the `_` prefix denotes that